test/web/controllers/questionnaire_page_test.exs

```diff
@@ -19,10 +19,15 @@ defmodule Advisor.Web.RequestPageTest do
              |> Enum.at(0)
              |> Floki.text == "Hello Felipe Sere!"
 
-    # maybe split this later into group_lead and advisor
+    number_of_group_leaders = 4
```

Comment:
just a really small thing... I think they're called 'group lead' instead of 'group leader'